dev-aws/kafka-shared-msk/pubsub/tf-kafka-config-admin.tf

```diff
@@ -0,0 +1,28 @@
+# Grant pubsub/tf-kafka-config-admin permission to all the resources
+resource "kafka_acl" "tf_kafka_config_admin_topic" {
+  resource_name       = "*"
+  resource_type       = "Topic"
+  acl_principal       = "User:CN=pubsub/tf-kafka-config-admin"
+  acl_host            = "*"
+  acl_operation       = "All"
+  acl_permission_type = "Allow"
```

Comment:
Is it possible to follow the principle of least privilege here, and provide an explicit list of exactly the operations we need to perform? Or do we really just need all of them all the time?

Reply:
We need them all